hack/checkout_account.py

```diff
@@ -36,22 +37,21 @@ def post_request(conn, input_state = "free"):
     })
 
     )
-    return conn.getresponse()
+    resp = conn.getresponse()
+    conn.close()
```

Comment:
I'm fairly certain that if you close the connection before reading the response (done below in `json.load(resp)`), that you won't get any of the data.

Reply:
arrgh, of course it would behave that way :joy: Let me fix up to return the json.